serverless.yml

```diff
@@ -40,6 +40,13 @@ functions:
           path: /login
           method: post
 
+  salt:
+    handler: com.javahelp.backend.endpoint.login.SaltHandler
+    events:
+      - httpApi:
+          path: /users/salt
+          method: get
+
```

Comment:
would it be better to pass in userid as a path param and have a separate path to fetch the salt via userid? something like /users/{userid}/salt?

Reply:
Another good point, ill add a new route